jupyter_notebooks/qancode_overview.ipynb

```diff
@@ -76,7 +76,7 @@
    },
    "outputs": [],
    "source": [
-    "qa = qancode.QANCODE(rc_url='https://v60rc1-master.demo.encodedcc.org/')"
+    "qa = qancode.QANCODE(rc_url='https://test.encodedcc.org')"
```

Comment:
Hmmm.... what do you think the best way to configure URL (or other input) for a jupytr note book?   Set them as constants at the top?  Some sort of form input?

Reply:
I would open overview notebook from a running notebook server and change in cell you are going to execute. Wouldn't mess with the raw _.ipynb_ contents you can see here or try to pass arguments from command line. QANCODE object stores URL so advantage of defining global variable at top that is only used once is negligible. This notebook only to demonstrate functionality in any case. You can see it rendered by nbviewer here: https://nbviewer.jupyter.org/github/ENCODE-DCC/pyencoded-tools/blob/master/jupyter_notebooks/qancode_overview.ipynb